internal/gatewayapi/runner/runner.go

```diff
@@ -110,11 +108,24 @@ func (r *Runner) startWasmCache(ctx context.Context) {
 		},
 		// Wasm cache options
 		wasm.CacheOptions{
-			CacheDir: wasmCacheDir,
+			CacheDir: wasmDir,
 		}, r.Logger)
 	r.wasmCache.Start(ctx)
 }
 
+func getWasmCacheDir() (string, error) {
+	homedir, err := os.UserHomeDir()
+	if err != nil {
+		return "", err
+	}
+	wasmDir := path.Join(homedir, ".eg", "wasm")
```

Comment:
can we `switch` this based on `GOOS` to minimize the change here ?

Reply:
if that works works for me too

Reply:
done